base/models.py

```diff
@@ -140,7 +162,11 @@ def firma(self, firmatario, concedi=True, modulo=None, motivo=None):
             raise ValueError("Il modulo richiesto per l'accettazione non e' stato completato correttamente.")
 
         self.concessa = concedi
-        self.firmatario = firmatario
+        if auto:
+            # TODO: decidere indirizzo email firmatario
+            self.firmatario = firmatario #settings.DEFAULT_FROM_EMAIL
```

Comment:
@fmarco fai un giro di controllo? @AlfioEmanueleFresta in assenza di firmatario si inserisce una dicitura che espliciti l'autorizzazione / negazione automatica?